MAUI/NumericEntry/Restriction.md

```diff
@@ -0,0 +1,94 @@
+---
+layout: post
+title: Value change restriction in .NET MAUI NumericEntry | Syncfusion
```

Comment:
title: Value Change Restriction in .NET MAUI NumericEntry | Syncfusion

Reply:
The requested content modification has been successfully addressed